ACHNBrowserUI/ACHNBrowserUI/views/turnips/charts/TurnipsChartBottomLegendView.swift

```diff
@@ -63,3 +78,22 @@ struct TurnipsChartBottomLegendView: View {
 private extension Int {
     var isAM: Bool { self == 0 || isMultiple(of: 2) }
 }
+
+struct TurnipsChartBottomLegendView_Previews: PreviewProvider {
+    static var previews: some View {
+        TurnipsChartBottomLegendView(predictions: predictions)
```

Comment:
You could directly use `TurnipsChartView_Previews.predictions` here, so you can avoid the code duplication of previews static values.

Reply:
Fixed in a commit.